packages/components/src/core/LoadingIndicator/index.tsx

```diff
@@ -8,13 +8,20 @@ import {
 export type LoadingIndicatorProps = Omit<
   RawLoadingIndicatorProps,
   "nonce" | "rev" | "rel" | "autoFocus" | "content"
->;
+> & {
+  "aria-label"?: string;
```

Comment:
Should we require this?  I guess that would be a breaking change.  I want people to fall into the pit of success though.

Reply:
Not a bad idea, although I think it's fine to have it be optional because there's a default of "Loading".

I put this in here in case someone wants to customize.